src/main/rx/rx.c

```diff
@@ -429,6 +429,11 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTime)
             auxiliaryProcessingRequired = true;
         }
     }
+
```

Comment:
Lines 424 / 425 are redundant now and should e removed.

Reply:
Sorry, missed that.  Updated and retested with a PPM receiver.